src/transformers/training_args.py

```diff
@@ -115,11 +133,13 @@ class TrainingArguments:
 
     @property
     def train_batch_size(self) -> int:
-        return self.per_gpu_train_batch_size * max(1, self.n_gpu)
+        per_device_batch_size = self.per_device_train_batch_size or self.per_gpu_train_batch_size
```

Comment:
we should log a warning if user uses `per_gpu_train_batch_size` (deprecation notice)